README.md

```diff
@@ -64,7 +74,7 @@ We also have official client-side implementations for
 [Java](https://github.com/metarhia/jstp-java)
 that work effortlessly on iOS and Android 🎉
 
-## Getting Started
+<h2 align="center">Getting Started</h2>
```

Comment:
@aqrln, I think this option will be better.
@metarhia/jstp-core, what's your opinion on this?